integration-tests/main/src/main/java/io/quarkiverse/googlecloudservices/it/BigtableResource.java

```diff
@@ -21,12 +20,10 @@
     private static final String TABLE_ID = "test-table";
     private static final String COLUMN_FAMILY_ID = "test-column-family";
 
-    @ConfigProperty(name = "quarkus.google.cloud.project-id")
-    String projectId;
-
     @PostConstruct
     void initBigtable() throws IOException {
-        try (BigtableTableAdminClient adminClient = BigtableTableAdminClient.create(projectId, INSTANCE_ID)) {
+        try (BigtableTableAdminClient adminClient = BigtableTableAdminClient.create(ServiceOptions.getDefaultProjectId(),
```

Comment:
For the integration tests, would it be preferable to still depend on `quarkus.google.cloud.project-id`? (i.e. would it still fallback to `ServiceOptions.getDefaultProjectId()`?) This may just keep the options open a little more if users wanted to override the default project in the integration tests for whatever reason.